includes/discovery/ports.inc.php

```diff
@@ -9,6 +9,9 @@
 $port_stats = snmpwalk_cache_oid($device, 'ifAlias', $port_stats, 'IF-MIB');
 $port_stats = snmpwalk_cache_oid($device, 'ifType', $port_stats, 'IF-MIB');
 $port_stats = snmpwalk_cache_oid($device, 'ifOperStatus', $port_stats, 'IF-MIB');
+if (Config::get('autodiscovery.xdp') === true) {
```

Comment:
Hi
It is not possible to run some xdp code here.

Reply:
Can't even see if it is enabled?

Reply:
Of course we can see, but it does not make sense in term of code structure. 
When you create modules to split different parts of the code, you just cannot do that. There are solutions to do this in a clean way, by storing the data in a dedicated xdp table (name to be defined) and populating it exclusively from the discovery_protocols module.